assets/js/util/index.js

```diff
@@ -445,20 +447,6 @@ export const getSiteKitAdminURL = ( page, args ) => {
 	return addQueryArgs( adminRoot, args );
 };
 
-/**
- * Verifies if the Front End site has been loaded in the iframe to check for tag presence.
- *
- * @return mixed Returns the iframe if it's loaded, false if not loaded.
- */
-export const isFrontendIframeLoaded = () => {
-	const iframe = document.getElementById( 'sitekit_fe_load_check' );
-	if ( iframe ) {
-		return iframe;
-	}
-
-	return false;
-};
-
```

Comment:
I see it imported and exported but not in use anywhere. Seems it was used in https://github.com/google/site-kit-wp/blob/6d344ce0c83560df41f6f8992a73b40271be68c4/tests/qunit/assets/js/util/index.js but but I'm not _exactly_ clear what it's trying to do. It doesn't seem to have been migrated over, perhaps because Jest isn't usually rendering full pages/browser DOM so it would have been harder to test?

Though the existing tests don't even seem to be testing many states 🤷‍♂. Seems safe to me to remove, but let me know if I'm off-base here.

Reply:
This is related to an older version of (existing) tag detection which used an iframe instead of `fetch`. This doesn't appear to be used anywhere like you said so it should be removed 👍